LLama/Experimental/Core/AntipromptStoppingCriteria.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LLama.Experimental.Abstractions;
+using LLama.Experimental.Common;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LLama.Experimental.Core
+{
+    // TODO: This is only the most simple implementation to run the test now. We should replace it in the future.
+    public class AntipromptStoppingCriteria: IStoppingCriteria
```

Comment:
This seems to be the same as the [`AntiPromptProcessor`](https://github.com/SciSharp/LLamaSharp/blob/f01c13ee54c5829180d546dd407fddf25e86cbe0/LLama/AntipromptProcessor.cs#L9)?

Reply:
Yes, but `IStoppingCriteria` is more flexible, which allows the sequence to be stooped because of text, token, length and any other reason (as long as the user could implement it).